resources/development-guidelines/code-of-conduct.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Code Of Conduct
+sidebar:
+  navigation:
+  - title: Resources
```

Comment:
I think we should do the same here as on `Payments` and not include `Resources` in the menu once you've navigated down into the `Development Guidelines` section. To go back to the top level of the `Resources` section, you would have to click the menu item in the top navigation bar, which is fine (for now).